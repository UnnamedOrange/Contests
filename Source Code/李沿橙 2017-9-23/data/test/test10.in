5
4
40 1 1 3 
50
6 227 529 728 467 10 406 426 840 616 952 949 900 26 349 586 388 117 944 222 623 147 640 992 342 570 916 372 932 556 269 740 344 558 161 972 54 2 528 462 224 752 336 137 826 704 152 452 220 298 
48
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 
50
1 933 743 262 529 700 508 752 256 256 119 711 351 843 705 108 393 330 366 169 932 917 847 972 868 980 223 549 592 164 169 551 427 190 624 635 920 944 310 862 484 363 301 710 236 876 431 929 397 675 
6
1 2 2 9 9 10 
