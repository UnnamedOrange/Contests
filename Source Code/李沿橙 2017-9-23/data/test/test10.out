1 1 3 40 
2 6 10 26 117 147 152 161 220 222 224 269 336 342 344 349 372 406 426 452 462 528 529 556 570 586 616 623 704 740 826 900 932 944 949 952 972 992 916 840 752 728 640 558 467 388 298 227 137 54 
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 2 
1 108 164 169 169 190 223 256 256 262 301 310 330 363 366 393 397 427 484 508 529 551 592 624 675 700 705 710 711 743 843 847 862 876 917 920 929 932 933 944 972 980 868 752 635 549 431 351 236 119 
1 2 9 10 9 2 
